application-admintools-default/src/main/java/com/xwiki/admintools/script/AdminToolsScriptService.java

```diff
@@ -64,6 +66,9 @@ public class AdminToolsScriptService implements ScriptService
     @Inject
     private Provider<XWikiContext> wikiContextProvider;
```

Comment:
It seems to me that you need the ``XWikiContext`` only to get the current wiki. You can get this from [``ModelContext``](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-api/src/main/java/org/xwiki/model/ModelContext.java) instead.